### ISSUE TITLE
Umake should list installed packages and their location.

### ISSUE THREAD

**Issue by jfdm** [Ubuntu/ubuntu-make#191]
`umake` should have an option to list the packages that are already installed, and also where they live.

I've noticed that `umake` doesn't sym-link binaries into `$HOME/.local/bin/`, and one has to link manually, or manually add paths to ones shell rc file. This, is more of an annoyance rather than an hindrance, and noticed in other reports that this is being /taken care of/. Being able to quickly work out what is installed and where it is installed would be beneficial. Especially is none default locations are used.

**Comment by didrocks** [comment]
Hey Jan. That's a very good point indeed!

I am still unsold on the `$HOME/.local/bin/` symlink as this one isn't added to the user's `$PATH`. That's why we add it on a per framework bases to `$PATH` for command line application as needed and provides `.desktop` files that integrates well with Unity launchers and other shells to point to the IDEs.
(this enables us to easily provide multiple flavors of pycharm and other ides like that).

However, as you formulated, having a way for the user to introspect and know what frameworks are installed is a really valuable easy feature to implement (we already have the frameworks and paths available in *~/.config/udtc*). So, I guess, we can get something like:
```
$umake list-installed
android-studio: /home/<user>/…
```

Also, we can have (even if `--help` provides that with additional info)
```
$ umake list
android-ndk
android-studio
pycharm
…
```

I would like to nominate this bug for new contributors first as it's an easy way to dive into the code, and eventually to Google Code-in.
Thanks for the suggestion!

**Comment by jfdm** [comment]
Cheers for the quick response. I've used make to install a few things, and glad it is here; installing recent versions of eclipse on an LTS were/is painful.

TBH I go either way wrt to installation location, and this issue is more about /Developer Experience/ and being able to see what I just installed and where it lives.

Although, this is outside the scope of this issue, I see `umake` as a hybrid combination of [GNU Stow](https://www.gnu.org/software/stow/) and [Homebrew](http://brew.sh/) and /Localised PPAs/ and [CheckInstall](https://help.ubuntu.com/community/CheckInstall). At somepoint it might be nice to see what can be borrowed from these projects to enhance the /Developer Experience/.
